rides/__init__.py

```diff
@@ -1,17 +1,24 @@
 ####################################
 # File name: __init__.py           #
 # Author: Ayush Goel & Fred Rybin  #
+# Contributors: Joe Abbate         #
 ####################################
 from subprocess import check_output
 import datetime
 import os
+from typing import TypedDict
+from flask_mail import Mail, Message
 import pytz
 from flask_pyoidc.flask_pyoidc import OIDCAuthentication
 from flask_pyoidc.provider_configuration import ProviderConfiguration, ClientMetadata
-from flask import Flask, render_template, send_from_directory, redirect, url_for, g
+from flask import Flask, render_template, send_from_directory, redirect, url_for, g, request, session
 from flask_sqlalchemy import SQLAlchemy
 from flask_wtf.csrf import CSRFProtect
 from flask_login import login_user, logout_user, login_required, LoginManager, current_user
+import pymysql
```

Comment:
Why is this necessary? I don't see it used anywhere?

Reply:
From what I researched online, Flask-SQLAlchemy can throw random errors in Python3. Pymysql just needs to be referenced there and in the URI and it does its job from what I saw.![image](https://user-images.githubusercontent.com/40615740/132513999-e9c2e9e8-95a2-4e0b-adb2-6809e2d7ffe2.png)

Reply:
I don't think that's the case based on the [flask docs](https://flask-sqlalchemy.palletsprojects.com/en/2.x/config/). Where is this stackoverflow post from?

Also, rideboard uses postgres in its full deployment.

Reply:
I don't have the link, it was what came up when I did the good ol' "Copy the error into google" technique. I'll try and run without it and see if it still works.

Reply:
Changed in commit https://github.com/ComputerScienceHouse/csh-rideboard/pull/22/commits/711d1a4ad688e98faa8a7ae76c4c812854517b00